spec/vehicle_spec.rb

```diff
@@ -14,7 +14,7 @@
       expect(@cruz.make).to eq('Chevrolet')
       expect(@cruz.model).to eq('Cruz')
       expect(@cruz.engine).to eq(:ice)
-      expect(@cruz.registration_date).to eq(nil)
+      expect(@cruz.registration_date).to eq(Date.today)
```

Comment:
One test that you are missing in this is for the `registration_date` method that you have.